Lib/idlelib/idle_test/test_squeezer.py

```diff
@@ -82,6 +92,15 @@ def test_several_lines_different_lengths(self):
 
 class SqueezerTest(unittest.TestCase):
     """Tests for the Squeezer class."""
+
+    @classmethod
+    def setUpClass(cls):
+        cls.font_family = get_font_family()
```

Comment:
This could also just be a global constant, i.e. set `FIXED_WIDTH_FONT_FAMILY = get_font_family()`.

Reply:
That would have to be wrapped in `try...except SkipTest...` to avoid skipping all non-gui tests.  As it is, the current line needs the same to avoid skipping non-gui tests within SqueezerTest.  But, looking ahead, I want to split SqueezerTest into SqueezerGuiTest and SqueezerNoGuiTest and move root creation into the GUI class setUpClass.  (This should also make it faster.)